BUILD.md

```diff
@@ -24,6 +24,7 @@ $ git clone --recurse-submodules https://github.com/rapidsai/cuml.git
 $ cd cuML
 $ mkdir build
 $ cd build
+$ export CUDA_BIN_PATH=$CUDA_HOME # (optional CUDA_HOME=/path/to/cuda/)
```

Comment:
This variable is only required if the cuda directory is not in the PATH env variable (or cmake doesn't like the cuda dir in the PATH for some reason), so I would add it as an optional step of the instructions